### PR TITLE
Add configbaker to the containerized development environment

### PR DESCRIPTION
**Pull request by GPortas**
## What this PR does / why we need it:

Adds configbaker to the containerized development environment, simplifying the bootstrapping process of the dataverse service container.

Edit (21/06): Fixed a versioning issue in a dependency of the design-system package, which is causing build issues on develop.

## Which issue(s) this PR closes:

N/A

## Special notes for your reviewer:

N/A

## Suggestions on how to test this:

Run the containerized development environment as usual

## Does this PR introduce a user interface change? If mockups are available, please link/include them here:

N/A

## Is there a release notes update needed for this change?:

N/A

## Additional documentation:

The README is updated with new instructions for running the environment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,13 +100,15 @@ Once the script has finished, you will be able to access Dataverse via:
 - [http://localhost:8000/spa](http://localhost:8000/spa): SPA Frontend
 - [http://localhost:8000](http://localhost:8000): Dataverse Backend and JSF Frontend
 
-If you want to run the environment with aggregated test data (collections and datasets), run the following command:
+_Note: The Dataverse configbaker takes some time to start the application, so the application will not be accessible until the bootstrapping is complete._
```
Comment (MellyGray) [inline README.md:103]: I ran the script without errors but I can only see the Payara home page running on localhost:8000, is there any additional step?

Comment (GPortas) [reply README.md:103]: Unfortunately we have to wait a few minutes until the configbaker bootstrapping process is finished. Please let me know if this time is excessive on your localhost or you find other kind of issues.

Comment (MellyGray) [reply README.md:103]: I tried again and it worked this time without waiting too much. Thanks!

**Review by MellyGray — APPROVED** [review]
Looks good to me!

**Pull request merged**